### PR TITLE
Fix/document type page

### PR DESCRIPTION
**Pull request by iNewLegend**

### CODE REVIEW

File: core/upgrade/upgrades.php [hunk 1]
```diff
@@ -51,7 +51,7 @@ public static function _v_0_3_2() {
 				continue;
 			}
 
-			$data = Plugin::$instance->db->iterate_data( $data, function( $element ) {
+			$data = Plugin::$instance->db->iterate_data( $data, function ( $element ) {
```
Comment (matipojo) [inline core/upgrade/upgrades.php]: Remove spaces here and below.

File: core/upgrade/upgrades.php [hunk 2]
```diff
@@ -555,4 +555,31 @@ public static function _v_2_6_6_fa4_migration_button( $updater ) {
 		Upgrade_Utils::_update_widget_settings( 'button', $updater, $changes );
 		Upgrade_Utils::_update_widget_settings( 'icon-box', $updater, $changes );
 	}
+
+	/**
+	 * Format was changed.
+	 */
+	public static function _v_2_6_7_remove_old_usage_data() {
```
Comment (matipojo) [inline core/upgrade/upgrades.php]: Move to tracking branch.

**Review by matipojo — CHANGES_REQUESTED** [review]

**Review by matipojo — CHANGES_REQUESTED** [review]
See the comments.

**Review by matipojo — APPROVED** [review]

File: core/upgrade/upgrades.php [hunk 3]
```diff
@@ -555,4 +555,23 @@ public static function _v_2_6_6_fa4_migration_button( $updater ) {
...
+		    	SELECT p1.ID FROM $wpdb->posts AS p LEFT JOIN $wpdb->posts AS p1 ON (p.ID = p1.post_parent || p.ID = p1.ID) WHERE p.post_type = 'page'
+			);
+		 ");
+
+		$wpdb->query( "UPDATE $wpdb->postmeta SET meta_value ='wp-post'
+			WHERE meta_key = '_elementor_template_type' && post_id in (
+		    	SELECT p1.ID FROM $wpdb->posts AS p LEFT JOIN $wpdb->posts AS p1 ON (p.ID = p1.post_parent || p.ID = p1.ID) WHERE p.post_type = 'post'
+			);
+		 ");
+	}
```
Comment (KingYes) [inline core/upgrade/upgrades.php]: Add call to `wp_cache_flush()`.

**Review by KingYes — CHANGES_REQUESTED** [review]

**Pull request merged**